Sources/BraintreePaymentFlow/Public/BraintreePaymentFlow/BTPaymentFlowClient.h

```diff
@@ -46,24 +44,13 @@ typedef NS_ENUM(NSInteger, BTPaymentFlowErrorType) {
  */
 - (void)onPaymentWithURL:(NSURL * _Nullable) url error:(NSError * _Nullable)error;
 
-/**
- Use when the payment flow was canceled.
- */
-- (void)onPaymentCancel;
```

Comment:
I think we still want to give merchants a hook into if the flow was cancelled

Reply:
I'm probably misunderstanding the intent of this method but it looks like this was called in `safariViewControllerDidFinish` which was an internal implementation delegate that was a part of `SFSafariViewController`. Was this exposed to merchants and just not used in our demo app? I also didn't see it in the docs so may just be unsure of how this was used previously.

Reply:
Ah - I see!

I was confused because this is in a public header, but you're right - it looks like `BTPaymentFlowDriverDelegate` only exists for internal conformance, not merchant implementation.

It looks like we return cancellations as Error types in the `startPaymentFlow` completion. 

It looks like it was made public, because it is used in the public header for `BTPaymentFlowClient.startPaymentFlow()`.

Reply:
The delegate on delegate on delegate pattern is confusing, so wanted to make sure I wasn't fully getting confused as well. 🙈